pybaseball/analysis/trajectories/utils.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+from functools import lru_cache
+from pybaseball.analysis.trajectories.unit_conversions import (
+    REVS_PER_MINUTE_TO_RADIANS_PER_SECOND,
+)
+
+
+@lru_cache(maxsize=64)
```

Comment:
Do we still need the LRU caches?

Reply:
we need it for `spin_components`, because we expect it to be called with the same values of `spin`, `spin_angle`, `launch_angle` and `launch_direction_angle`. i.e. wx, wy, wz are based on the initial spin and angles, and don't change throughout the computation of the trajectory. as with the celsius to kelvin conversion, this is because it's how it's done in Alan Nathan's code and I just transcribed it.

we don't need it for `initial_velocity`